src/libs/Launcher.functions.ts

```diff
@@ -0,0 +1,27 @@
+import CozyClient, { Q } from 'cozy-client'
+import { CozyClientDocument } from 'cozy-client/types/types'
+
+export const cleanExistingAccountsForKonnector = async (
+  client: CozyClient,
+  konnectorSlug: string,
+  log?: MiniLogger
```

Comment:
Why did you make `log` optional?

Reply:
Didn't want to inject it during the test as I didn't care if it worked or not, so I figured might as well make it optional. The function works whether you inject log or not anyway and is not a logging function per se, it's not tested because TypeScript enforces the verification